PlacesOfInterest/app/src/main/java/com/gmail/petraccaro/angelo/placesofinterest/LoginActivity.java

```diff
@@ -81,11 +80,11 @@ else if (!isEmailValid(email)) {
             mEmailView.setError(getString(R.string.error_invalid_email));
         }
 
-        else {
+        else {*/
             FirebaseAuth mAuth = FirebaseAuth.getInstance();
             mAuth.signOut();
 
-            mAuth.signInWithEmailAndPassword(email,password)
+            mAuth.signInWithEmailAndPassword("antonio@gmail.com","ciaociao")

```

Comment:
Change this to an input login from the user

Reply:
ok